### ISSUE TITLE
Gildengefechte Status als csv

### ISSUE THREAD

**Issue by Harr235** [mainIine/foe-helfer-extension#2009]
Hallo, ich fände es hilfreich, wenn ich bei den Gildengefechte die Daten als CSV herunterladen könnte. 

LG Harry

**Comment by outoftheline** [comment]
Meinst du das hier?
![image](https://user-images.githubusercontent.com/7482863/118977639-ab879100-b976-11eb-840b-186252f5cc24.png)

**Comment by Harr235** [comment]
nein, sondern die Übersicht über die aktuellen und nächsten Sektoren. Ich hoffe, du weißt, was ich meine. Leider kann ich gerade kein Bild einstellen, da ich Gildenwechsel hatte und erst nächste Woche wieder dabei bin.

**Comment by outoftheline** [comment]
Ja, verstehe was du meinst. Aber wieso? :D

**Comment by Harr235** [comment]
um die Daten schneller in Excel bearbeiten zu können und dann in den Taktik-Chat einzustellen.
